### DISCUSSION TITLE
Where is the executable folder after compiling using `cargo` command for the example game ?

### DISCUSSION THREAD

**Discussion by AnotherCoder1** [bevyengine/bevy#9224] [Q&A] [answered]
after typing `cargo run --example breakout` the cargo compiles the source code well with the necessary library and the game is spawned.

But where is the executable file or .exe file located?

I am on Windows by the way.

**Comment by nicopap** [comment, marked as answer]
When you do `cargo run --example` it should print in the terminal something along the lines of

```
Running `/home/code/bevy/target/debug/bevypunk`
```

This should show the path to the executable cargo is running.

It's the last row in the long column of text that goes "Compiling …" (in green) after you run the command but before the game runs.

**Reply by AnotherCoder1** [reply]
ok thank you for the help.
